src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -22,7 +22,7 @@
       %li.nav-item.scrollable-tab-link
         = link_to('Mentioned Issues', '#mentioned-issues', class: 'nav-link text-nowrap', 'aria-controls': 'mentioned-issues',
                   'aria-selected': 'false', 'data-toggle': 'tab', role: 'tab')
-    .tab-content#request-tabs-content
+    .tab-content.p-4#request-tabs-content
```

Comment:
There is no padding because you directly render some unformated test into a tab....

```
cat -p app/views/webui/request/beta_show_tabs/_overview.html.haml 
Overview
```

Reply:
Paragraphs or any other [block-level elementss](https://www.w3schools.com/html/html_blocks.asp) usually have padding.

Reply:
> Paragraphs or any other [block-level elementss](https://www.w3schools.com/html/html_blocks.asp) usually have padding.

Yes, they usually do. On OBS though, paragraphs don't have any padding. This is why we need to apply padding to the container.

Reply:
Still the overview partial should not just be inline text

Reply:
This is only a temporary placeholder. This will change in the upcoming days.